addon/manifest.json

```diff
@@ -8,7 +8,7 @@
       "update_url": "https://testpilot.firefox.com/files/@testpilot-addon/updates.json"
     }
   },
-  "version": "3.0.0",
+  "version": "3.0.0vffeb6433",
```

Comment:
Looks like the altered version snuck into the commit - we should change this back to 3.0.0 so it doesn't end up on production

Reply:
> so it doesn't end up on production

you mean like production right now? ;)

<img width="449" alt="screen shot 2018-04-16 at 10 49 10 am" src="https://user-images.githubusercontent.com/87619/38826167-82e185f8-415b-11e8-8da0-e8123247b3f5.png">

We should have a build directory since we have build time assets like this

Reply:
Yeah 😞 I forget when that broke, I think sometime when we moved to mozilla key signing or circle 2.0. Also, yeah, this script should probably be reworked to not alter the in-repo package or manifest